### ISSUE TITLE
(Bug): averageRating is missing

### ISSUE THREAD

**Issue by surrealegg** [Anarios/return-youtube-dislike#301]
### Browser

Microsoft Edge

### Browser Version

96.0.1054.53

### Extension or Userscript?

Extension

### Extension/Userscript Version

1.1.0.2

### Video link where you see the problem

https://www.youtube.com/watch?v=kxOuG8jMIgI

### What happened?

Seems like YouTube removed "averageRating" property. 
![Search](https://i.imgur.com/vvAMf6v.png)
```
[return youtube dislike]: response from api:
[return youtube dislike]: {"id":"kxOuG8jMIgI","dateCreated":"2021-12-14T08:39:37.828167Z","likes":26835,"dislikes":476722,"rating":1.2131635544734758,"viewCount":3086951,"deleted":false}
[return youtube dislike]: response from youtube:
[return youtube dislike]: {"likes":26835,"dislikes":null,"viewCount":3087098}
```
So because of this, dislike count is overwritten to 0.
![Preview](https://i.imgur.com/ZWgBtJu.png)

**Comment by Anarios** [comment]
Well, average rating missing is not our bug :)  Latest version available is 2.0.0.1 - everything works there.

**Comment by surrealegg** [comment]
Oops, my mistake! I forgot to update 😅
